src/js/components/bulkDownload/awards/filters/dateRange/TimePeriodFilter.jsx

```diff
@@ -247,7 +247,8 @@ export default class TimePeriodFilter extends React.Component {
                         hideError={this.hideError} />
                     { errorDetails }
                     <p className="data-note">
-                        Note: data is available for download from 2001 - present
+                        Note: Note: data is available for download from FY 2001 - present.
+                        To select data prior to FY 2009, enter an appropriate date range. Date ranges may span up to one year.
```

Comment:
Instead of hard-coding `FY 2009`, we should display the earliest FY that there exists a button for. E.g., when FY 2019 data is available, the options will shift to FY 2010 - FY 2019.